modules/automationAccount.bicep

```diff
@@ -211,7 +213,7 @@ resource runCommand 'Microsoft.Compute/virtualMachines/runCommands@2023-07-01' =
       }
       {
         name: 'StorageAccountName'
-        value: storageAccountName
+        value: split(storageAccountResourceId, '/')[8]
```

Comment:
My only comment on splitting, is that it's less readable in my opinion for a new user of the code or maybe a new person to bicep/PowerShell to understand what this value is derived from...not as straight forward.

Reply:
I was trying to reduce the number of params I'm passing around and it seemed unnecessary since I had the resource ID anyway. We can always add it back, but it seems like bloat to me.  Maybe a comment would be better? Thoughts?